.one-pipeline.yaml

```diff
@@ -114,6 +130,79 @@ containerize:
   image: icr.io/continuous-delivery/pipeline/pipeline-base-image:2.12@sha256:ff4053b0bca784d6d105fee1d008cfb20db206011453071e86b69ca3fde706a4
   script: |
     #!/usr/bin/env bash
+
+    PERIODIC_TRIGGER_RUN=$(get_env periodic-trigger-run)
+    PERIODIC_TRIGGER_RUN="$(echo "$PERIODIC_TRIGGER_RUN" | tr '[:upper:]' '[:lower:]')"
+    #  Build images
+    export PIPELINE_USERNAME=$(get_env ibmcloud-api-user)
+    export PIPELINE_PASSWORD=$(get_env ibmcloud-api-key-staging)
+    export PIPELINE_REGISTRY=$(get_env pipeline-registry)
+    export PIPELINE_PRODUCTION_IMAGE=$(get_env pipeline-production-image)
+    export PIPELINE_OPERATOR_IMAGE=$(get_env pipeline-operator-image)
+    export DOCKER_USERNAME=$(get_env docker-username)
+    export DOCKER_PASSWORD=$(get_env docker-password)
+    export REDHAT_USERNAME=$(get_env redhat-user-id)
+    export REDHAT_PASSWORD=$(get_env redhat-password)
+    export REDHAT_BASE_IMAGE=$(get_env redhat-base-image)
+    export REDHAT_REGISTRY=$(get_env redhat-registry)
+    export OPM_VERSION=$(get_env opm-version)
+    export DISABLE_ARTIFACTORY=$(get_env disable-artifactory)
+
+     # Download and install skopeo
+    if ! command -v skopeo &> /dev/null; then
+      PACKAGE_CACHE_URL=$(get_env package-cache-url)
+      wget --no-verbose "$PACKAGE_CACHE_URL/skopeo_1.2.2-2_amd64.deb"
+      wget --no-verbose "$PACKAGE_CACHE_URL/containers-common_1-22_all.deb"
+      wget --no-verbose http://archive.ubuntu.com/ubuntu/pool/main/g/gpgme1.0/libgpgme11_1.10.0-1ubuntu1_amd64.deb
+      sudo dpkg -i libgpgme11_1.10.0-1ubuntu1_amd64.deb
+      sudo dpkg -i containers-common_1-22_all.deb
+      sudo dpkg -i skopeo_1.2.2-2_amd64.deb
+    fi
+    skopeo --version || exit 1
+
+    if [[ ! -z "$PERIODIC_TRIGGER_RUN" && "$PERIODIC_TRIGGER_RUN" != "false" && "$PERIODIC_TRIGGER_RUN" != "no"  ]]; then
+      echo "Skipping containerize, but generating list of images. This is a periodic run that is only meant to produce CVE information."
+      RELEASE_TARGET=$(curl --silent "https://api.github.com/repos/WASdev/websphere-liberty-operator/releases/latest" | jq -r .tag_name)
```

Comment:
It's unlikely we would have to rescan a release prior to latest. But it'll be good to have the option to specify the release target via some pipeline config option.